Added Logs To Sell Vehicle To Customer

djontop opened the pull request:
Here We Added Logs To Sell Vehicle To Customers. When Employee Sell Vehicle To Customer Log Will Be Created ❣️
![image](https://user-images.githubusercontent.com/107875648/235548214-2d4a07c4-a332-4db7-b103-1726a89b9b8a.png)

djontop committed:
Added Logs To Sell Vehicle To Customer

Here We Added Logs To Sell Vehicle To Customers. When Employee Sell Vehicle To Customer Log Will Be Created ❣️

djontop committed:
commission go to boss menu & Logs To Sell Vehicle

Added commission go to boss menu + Added Logs To Sell Vehicle To Customers. Added Logs To Sell Vehicle To Customers. When Employee Sell Vehicle To Customer Log Will Be Created ❣️

djontop committed:
Added Config for boss menu commission

Added Config for boss menu commission. From They Can Change percentage if commission goes to boss menu  
Config.bossmenu = 0.50 -- Percent that goes to Boss Menu / qb-management from a full car sale 50%

djontop commented:
Personally Verified Everything ❣️

GodyFromDiscord commented:
A log for this already exists under qb/smallresources/server/logs.lua 

    ['vehicleshop'] = '',

djontop commented:
> vehicleshop

Its Different I Added Logs To Vehicle Shop Not Vehicle Sale. log Added in qb/smallresouces/server/logs is of vehiclesale with is when player sale his owned vehicle that logs created . But I added Logs to When you purchase vehicle from PDM From sales man

gigitygamer commented on server.lua:
```diff
@@ -306,6 +306,7 @@ RegisterNetEvent('qb-vehicleshop:server:sellShowroomVehicle', function(data, pla
         local vehicle = data
         local vehiclePrice = QBCore.Shared.Vehicles[vehicle]['price']
         local commission = round(vehiclePrice * Config.Commission)
+        local bossmenu = round(vehiclePrice * Config.Commissionboss)
```
This needs to be local bossmenu = round(vehiclePrice * Config.bossmenu) - per your changes to the config.lua in this pull
